utils/webhook_cache.py

```diff
@@ -17,31 +17,31 @@ def store_webhooks(self, webhooks: list, identifiers: list, servers: list):
             webhook = webhooks[index]
             identifier = identifiers[index]
             server = servers[index]
-            if not server in self.__webhooks.keys():
-                self.__webhooks.update({server: {identifier: webhook}})
-            self.__webhooks[server].update({identifier: webhook})
+            self.__webhooks.setdefault(server, {})[identifier] = webhook
         return len(self.__webhooks)
 
-    def get_webhooks(self, server: int or str):
+    def get_webhooks(self, server: Union[int, str]):
         try:
             server = int(server)
         except:
             pass
-        if len(self.__webhooks[server].values())==0:
+
+        if len(self.__webhooks[server].values()) == 0:
             raise ValueError('no webhooks')
         return list(self.__webhooks[server].values())
 
-    def get_webhook(self, identifier: int or str):
+    def get_webhook(self, identifier: Union[int, str]):
```

Comment:
```suggestion
    def get_webhook(self, identifier: int | str):
```
Once again not really required